dces-drc-integration/src/main/java/uk/gov/justice/laa/crime/dces/integration/model/external/ErrorDTO.java

```diff
@@ -0,0 +1,10 @@
+package uk.gov.justice.laa.crime.dces.integration.model.external;
+
+/**
+ * Represents an error response from the MAAT court data API.
+ *
+ * @param code can be one of `BAD_REQUEST`, `NOT_FOUND`, `DB Error`, `Object Not Found`, `Application Error`.
+ * @param message human-readable error message.
+ */
+public record ErrorDTO(String code, String message) {
```

Comment:
Is there any value in the code parameter being an enum if the code is limited to only 4 possible values.

Reply:
This is a DTO that has to match a type in the MAAT court data API repository, so it has to be `String` (since it is a `String` there, and changing it in MAAT API would be a huge change - also, those are just the values it takes on for our services: other services might use other codes).